database/firebird/firebird_test.go

```diff
@@ -38,8 +38,9 @@ var (
 		},
 	}
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "jacobalberty/firebird:2.5-ss", Options: opts},
```

Comment:
Confirming that v2.5 is no longer supported:
https://www.firebirdsql.org/en/discontinued-versions/